### PR TITLE
Palindrome Number(PYTHON) 

### PR DESCRIPTION
**Pull request by Srishti013**
Fixes[#1660]

### CODE REVIEW

**Comment by Srishti013** [comment]
Please check  @rish-singhal @Rohan27s

**Comment by Srishti013** [comment]
Please check @rish-singhal @Rohan27s @rudrakshi99 @tarun26091999 @nakul-19 @OjusWiZard @yasharth291

**Comment by Srishti013** [comment]
@rish-singhal please review

File: Algorithm/Number_theory/readme.md [hunk 1]
```diff
@@ -61,6 +58,7 @@
 - Check Trimorphic number ---->[C++](/Code/C++/Trimorphic_number.cpp)
 - Fibonacci Series ----> [Java](/Code/Java/Fibonacci_Series.java)
 - GCD of two number using Euclidean Algorithm ----> [Java](/Code/Java/GCD.java) | [Python](/Code/Python/GCD.py)
+- Palindrome Number --->[Python](/Code/Python/palindrome_number.py)
 - Taylor's Series of e^x ----> [Java](/Code/Java/Taylorseries.java)
```
Comment (rish-singhal) [inline Algorithm/Number_theory/readme.md:64]: Sort this list.

**Review by rish-singhal — CHANGES_REQUESTED** [review]

**Comment by Srishti013** [comment]
Done @rish-singhal

**Pull request merged**